certbot/docs/install.rst

```diff
@@ -6,80 +6,49 @@ Get Certbot
    :local:
 
 
-About Certbot
-=============
 
-*Certbot is meant to be run directly on a web server*, normally by a system administrator. In most cases, running Certbot on your personal computer is not a useful option. The instructions below relate to installing and running Certbot on a server.
+.. _system_requirements:
 
-System administrators can use Certbot directly to request certificates; they should *not* allow unprivileged users to run arbitrary Certbot commands as ``root``, because Certbot allows its user to specify arbitrary file locations and run arbitrary scripts.
+System Requirements
+===================
 
-Certbot is packaged for many common operating systems and web servers. Check whether
-``certbot`` (or ``letsencrypt``) is packaged for your web server's OS by visiting
-certbot.eff.org_, where you will also find the correct installation instructions for
-your system.
+- Python 3.7+
+- UNIX-like operating system
+- Root access
+- Port 80 Open
 
-.. Note:: Unless you have very specific requirements, we kindly suggest that you use the installation instructions for your system found at certbot.eff.org_.
+.. Note:: To run without root privileges, but for most users who want to avoid running an ACME client as root, either letsencrypt-nosudo or simp_le are more appropriate choices.
```

Comment:
The advice here needs a refresh. The projects mentioned look abandoned and aren't going to be helpful to users.

What might be useful to mention to users is that Certbot is most useful when run with root privileges, because it is then able to automatically configure SSL for Apache and nginx.